incubator/hnc/test/e2e/demo_test.go

```diff
@@ -87,4 +92,37 @@ var _ = Describe("Demo", func() {
 		RunShouldContain(nsService1, 2, "kubectl hns describe", nsTeamA)
 		RunShouldNotContain("my-creds", 2, "kubectl -n", nsService1, "get secrets")
 	})
+
+	It("Should create and delete subnamespaces", func(){
+		MustRun("kubectl create ns", nsOrg)
+		MustRun("kubectl hns create", nsTeamA, "-n", nsOrg)
+		MustRun("kubectl hns create", nsService1, "-n", nsTeamA)
+		MustRun("kubectl hns create", nsService2, "-n", nsTeamA)
+		MustRun("kubectl hns create", nsService3, "-n", nsTeamA)
+		RunShouldContain(nsTeamA+"\n├── "+nsService1+"\n├── "+nsService2+"\n└── "+nsService3, 2, "kubectl hns tree", nsTeamA)
+		MustRun("kubectl hns create", nsDev, "-n", nsService1)
+		RunShouldContain(nsDev, 2, "kubectl hns describe", nsService1)
```

Comment:
The first param here should probably be more descriptive - eg something like `"SubnamespaceOf: "+nsdev` or whatever the actual text is.

Reply:
done